.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+/.bundle/ @cockroachdb/docs-infra-prs
+/.github/ @cockroachdb/docs-infra-prs
+/__tests__/ @cockroachdb/docs-infra-prs
+/_layouts/  @cockroachdb/docs-infra-prs
+/_plugins/  @cockroachdb/docs-infra-prs
+/archived/  @cockroachdb/docs-infra-prs
+/css/ @cockroachdb/docs-infra-prs
+/fonts/ @cockroachdb/docs-infra-prs
+/githooks/  @cockroachdb/docs-infra-prs
+/jekyll-algolia-dev/  @cockroachdb/docs-infra-prs
+/js/  @cockroachdb/docs-infra-prs
+/netlify/ @cockroachdb/docs-infra-prs
+/vale/  @cockroachdb/docs-infra-prs
+_redirects  @cockroachdb/docs-infra-prs
+.gitattributes  @cockroachdb/docs-infra-prs
+.gitignore  @cockroachdb/docs-infra-prs
+.htmltest.yaml  @cockroachdb/docs-infra-prs
+.vale.ini @cockroachdb/docs-infra-prs
+Gemfile @cockroachdb/docs-infra-prs
+LICENSE @cockroachdb/docs-infra-prs
```

Comment:
I don't think this should be on `docs-infra-prs`. Maybe a different group for docs management—but I'd leave this out for now if you are OK with that. Let me know if you disagree.

Reply:
That's fair. I'll remove.